python/dask_cudf/dask_cudf/core.py

```diff
@@ -421,7 +425,7 @@ def _naive_var(ddf, meta, skipna, ddof, split_every, out):
 def _parallel_var(ddf, meta, skipna, split_every, out):
     def _local_var(x, skipna):
         if skipna:
-            n = x.count(skipna=skipna)
```

Comment:
Hmm, did `count` ever accept `skipna`?

Reply:
Yup, `count` never had `skipna` and `count` only returns the count of non-NA values.